package.json

```diff
@@ -37,13 +37,15 @@
     "angular-ui-ace": "^0.2.3",
     "angular-ui-bootstrap": "^2.5.0",
     "angular-vs-repeat": "^1.1.7",
+    "angularjs-dragula": "^2.0.0",
     "bootstrap": "^3.3.7",
     "brace": "^0.10.0",
     "core-js": "https://registry.npmjs.org/core-js/-/core-js-2.4.1.tgz",
     "cornelius": "git+https://github.com/restorando/cornelius.git",
     "d3": "^3.5.17",
     "d3-cloud": "^1.2.4",
     "debug": "^3.1.0",
+    "dragula": "^3.7.2",
```

Comment:
Did you see we already have [sortable](https://github.com/getredash/redash/blob/master/client/app/lib/sortable.js)? I don't mind switching, but prefer to keep only one lib for the same functionality.

Reply:
Missed it. Will remove dragula and use it.